dbms/src/Flash/Coprocessor/AggregationInterpreterHelper.cpp

```diff
@@ -31,6 +30,18 @@ bool isFinalAggMode(const tipb::Expr & expr)
     return expr.aggfuncmode() == tipb::AggFunctionMode::FinalMode || expr.aggfuncmode() == tipb::AggFunctionMode::CompleteMode;
 }
 
+bool isSumOnPartialResults(const tipb::Expr & expr)
+{
+    if (!expr.has_aggfuncmode())
+        return false;
+
+    if (getAggFunctionName(expr) == "sum" && expr.aggfuncmode() == tipb::AggFunctionMode::FinalMode)
```

Comment:
According to [here](https://github.com/pingcap/tidb/blob/9c0d6157e2c787739eaf84bbc112c944b666302b/expression/aggregation/aggregation.go#L99), it should return true for both `FinalMode` and `Partial2Mode`

Reply:
> According to [here](https://github.com/pingcap/tidb/blob/9c0d6157e2c787739eaf84bbc112c944b666302b/expression/aggregation/aggregation.go#L99), it should return true for both `FinalMode` and `Partial2Mode`

okk